kore/test/Test/Kore/Repl/Parser.hs

```diff
@@ -190,6 +192,31 @@ redirectTests =
     , "config 5 > file" `parsesTo_` Redirect (ShowConfig (Just 5)) "file"
     , "config 5 > file" `parsesTo_` Redirect (ShowConfig (Just 5)) "file"
     , "claim 3 > cf"    `parsesTo_` Redirect (ShowClaim 3)         "cf"
+    , "config 5 > "      `fails`     "no file name"
+    ]
+
+pipeTests :: [ParserTest ReplCommand]
+pipeTests =
+    [ "config | script"             `parsesTo_` Pipe (ShowConfig Nothing) "script" []
```

Comment:
For this test and the next one, it might be easier to write a helper constructor function to make things fit nicer.